pages/api/paybutton/transactions/[id].ts

```diff
@@ -0,0 +1,30 @@
+import { RESPONSE_MESSAGES } from 'constants/index';
+import { NextApiRequest, NextApiResponse } from 'next'
+import { fetchTransactionsByPaybuttonId } from 'services/transactionService'
+
+export default async ( req: NextApiRequest, res: NextApiResponse): Promise<void> => {
+  if (req.method === 'GET') {
```

Comment:
We should check if the user requesting the txs is indeed the owner of the paybutton. Or if it's an admin.

Check `pages/api/paybutton/[id].ts` for an example of that being done.